eZ/Publish/Core/FieldType/Url/UrlStorage/Gateway/DoctrineStorage.php

```diff
@@ -129,6 +129,48 @@ public function insertUrl($url)
         );
     }
 
+    /**
+     * Return a list of URLs used by the given field and version.
+     *
+     * @return bool[] An array of URLs, with urls as keys
```

Comment:
This type specification is not correct. ATM you're returning `array<string, boolean>`